src/scripts.js

```diff
@@ -192,7 +226,7 @@ function showSavedRecipes() {
 // CREATE RECIPE INSTRUCTIONS
 function openRecipeInfo(event) {
   fullRecipeInfo.style.display = "inline";
-  let recipeId = event.path.find(e => e.id).id;
+  let recipeId = event.composedPath().find(e => e.id).id;
```

Comment:
Would love to go over .path vs. composedPath a little more for group clarity

Reply:
Yeah, same here. I kinda get it, but I think a quick read up on it might make things more clear. I think it depends on the browser too

Reply:
Yes, let's learn more about path vs composedPath().